custom/icds_reports/utils/aggregation_helpers/monolith/agg_awc.py

```diff
@@ -363,14 +364,16 @@ def updates(self):
                 sum(due_list_ccs) AS usage_num_due_list_ccs,
                 sum(due_list_child) AS usage_num_due_list_child_health
             FROM "{usage_table}"
-            WHERE month = %(start_date)s GROUP BY awc_id, month
+            WHERE month = %(start_date)s AND form_date >= %(start_date)s AND form_date < %(next_month)
```

Comment:
I intentionally did not change this, for the distributed query as month has an index on it and we don't have partitions so `form_date` filtering would have no effect